patches/net-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/net/BUILD.gn b/net/BUILD.gn
-index 40f143dd9ba73d4fa17f6fa512f2af7a7d6b5d1d..7f7a09d4b0dedb130390396c8337a836697a263e 100644
+index 40f143dd9ba73d4fa17f6fa512f2af7a7d6b5d1d..dd9c8882e8db6510e2f710eb76595d633136f555 100644
 --- a/net/BUILD.gn
 +++ b/net/BUILD.gn
 @@ -1131,6 +1131,7 @@ component("net") {
        "reporting/reporting_uploader.h",
      ]
    }
-+  sources += brave_net_sources
++  import("//brave/net/sources.gni") sources += brave_net_sources deps += brave_net_deps
```

Comment:
you don't need to write add `import("//brave/net/sources.gni")` here, because `brave_net_sources` is already accessible via `//brave/build/config/brave_build.gni`.